pyaedt/generic/near_field_import.py

```diff
@@ -0,0 +1,164 @@
+import os
+import re
+import csv
+import glob
+
+
+class BoxFacePointsAndFields(object):  # pragma: no cover
```

Comment:
Both this class and `convert_nearfield_data` the next should have coverage.

Reply:
good catch. Thanks I've forgot to remove once I've added UT.